barretenberg/cpp/src/barretenberg/translator_vm/relation_correctness.test.cpp

```diff
@@ -538,6 +538,7 @@ TEST_F(TranslatorRelationCorrectnessTests, Decomposition)
         prover_polynomials, params, "TranslatorDecompositionRelation");
 }
 
+// This test just needs fixing the relation should be fine
```

Comment:
?

Reply:
leftover comment to myself as I was debugging :D